### ISSUE TITLE
Encoding for messages sent to Flowdock broken

### ISSUE THREAD

**Issue by mrnugget** [applikatoni/applikatoni#25]
Whenever you use a character like `'` in a deployment comment and the comment is sent to Flowdock the encoding is broken:

![2015-12-14 at 14 32](https://cloud.githubusercontent.com/assets/1185253/11782156/926ef2d6-a26f-11e5-92d3-ae43fa57e0ca.png)

In this example `"` gets turned into `&#34;`.

The complete code for the Flowdock notifier [can be found here](https://github.com/applikatoni/applikatoni/blob/3abe9913cd866cd71f0ddd38001a46a733531483/server/flowdock_notifier.go).

I'll see what I can dig up in regards to the encoding and the Flowdock API.

**Comment by mrnugget** [comment]
The problem is definitely on our side:

```go
package main

import (
	"bytes"
	"fmt"
	"html/template"
)

var tmplStr = `{{.Comment}}`

var tmpl = template.Must(template.New("test").Parse(tmplStr))
var result bytes.Buffer

func main() {
	err := tmpl.Execute(&result, map[string]interface{}{
		"Comment": `this is a "comment" with "quotes" and 'single quotes'`,
	})
	if err != nil {
		panic(err)
	}

	fmt.Printf("result: %q\n", result.String())
}
// outputs:
// result: "this is a &#34;comment&#34; with &#34;quotes&#34; and &#39;single quotes&#39;"
```
[Link to Go Playground](http://play.golang.org/p/fvuxNB0qPw)

**Comment by mrnugget** [comment]
Yep, confirmed: the bug doesn't exist in `1.2.0` and was fixed in #24. The build the bug report was based on didn't include these changes.

Now it works fine:

![2015-12-15 at 08 42](https://cloud.githubusercontent.com/assets/1185253/11804918/df8022be-a307-11e5-9b43-f14049d1e846.png)

**Comment by Wind0r** [comment]
and I only hoped the change to text/template wouldn't break any wanted 'html' feature.
Even better it fixed a bug :pizza:
